### PR TITLE
Added everything

### PR DESCRIPTION
**Pull request by pabbathisowmya**
updated

### CODE REVIEW

**Comment by Dhruv1064** [comment]
Add the cost structure in both readme file and report

File: README.md [hunk 1]
```diff
@@ -1,2 +1,121 @@
-# Hop-Bot
-Srishti 2020
+# HopBot
```
Comment (SanjeevKrishnan) [inline README.md:1]: Make it Hop-Bot. And add "Srishti 2020" below it.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,121 @@
...
+
+##  ABSTRACT:
+ Hop bot is used to jump over any obstacle of height of comparable dimension to that of the bot. Motor, Springs and Gears are used for its proper functioning along with a pair of Non-motorized wheels.
+
+![Image](https://github.com/pabbathisowmya/HopBot/blob/master/Images/Mechanical%20cad.jpg)
+
+## INTRODUCTION:
```
Comment (SanjeevKrishnan) [inline README.md]: This should be "Motivation", not "Introduction"

File: README.md [hunk 3]
```diff
@@ -1,2 +1,121 @@
...
+
+### ACCELEROMETER:
+#### _Motto of usage:_
+ It is electromechanical device and was used to sense the vibration, orientation and acceleration of a bot.
+
+### GYROSCOPE:
+#### _Motto of usage:_
+ Gyroscope was used to determine angular velocity orientation of bot.
+
+### WORKFLOW:
```
Comment (SanjeevKrishnan) [inline README.md]: Move this Workflow above Mechanical Aspect. Also "Workflow",  "Applications", "Limitations", "Future Improvement", "Team Members", "Mentors", "Refernce" have to be H2 type heading. For example `##Workflow`, not `###Workflow`.

File: README.md [hunk 4]
```diff
@@ -1,2 +1,121 @@
...
+
+![Image](https://github.com/pabbathisowmya/HopBot/blob/master/Images/Work%20Flow.PNG)
+
+### APPLICATIONS:
+ Hop bot has a strong ability to overcome obstacles. It can be applied to the occasion with complex and changeable environment, such as detection of plane surface, post disaster relief and military reconnaissance.
+
+### FUTURE IMPROVEMENTS:
```
Comment (SanjeevKrishnan) [inline README.md]: Add Limitations after Applications

File: README.md [hunk 5]
```diff
@@ -1,2 +1,121 @@
...
+### GYROSCOPE:
+#### _Motto of usage:_
+ Gyroscope was used to determine angular velocity orientation of bot.
+
+### WORKFLOW:
+
+![Image](https://github.com/pabbathisowmya/HopBot/blob/master/Images/Work%20Flow.PNG)
+
+### APPLICATIONS:
+ Hop bot has a strong ability to overcome obstacles. It can be applied to the occasion with complex and changeable environment, such as detection of plane surface, post disaster relief and military reconnaissance.
```
Comment (SanjeevKrishnan) [inline README.md]: Applications has to be in list form

File: README.md [hunk 6]
```diff
@@ -1,2 +1,121 @@
...
+
+### FUTURE IMPROVEMENTS:
+ 1. Changes can be made to move it in a desirable direction.
+ 2. Jump height can be increased by using a strong motor. 
+ 3. Time interval between two simultaneous jumps can be decreased.
+
+### TEAM MEMBERS:
+ 1.	[Anshul saini](https://github.com/Anshulsaini24)
```
Comment (SanjeevKrishnan) [inline README.md:179]: Team Members have to be in alphabetical order.

File: README.md [hunk 7]
```diff
@@ -1,2 +1,121 @@
...
+ 5.	[R. pranav](https://github.com/Pranavravichandran)
+ 6.	[Jahnavi Tarale](https://github.com/janhavi729)
+
+### MENTOR:
+ 1.	[Dhruv seghal](https://github.com/Dhruv1064)
+ 2. [Suja](https://github.com/suja-g)
+ 3. [Virendra Yadav](https://github.com/virenyadav01)
+
+### REFERENCE:
+ 1. [https://www.hindawi.com/journals/abb/2017/4780160/]
```
Comment (SanjeevKrishnan) [inline README.md]: Remove square brackets from URL.

File: README.md [hunk 8]
```diff
@@ -1,2 +1,121 @@
...
+![Image](https://github.com/pabbathisowmya/HopBot/blob/master/Images/Arduino%20uno.jpg)
+
+### ACCELEROMETER:
+#### _Motto of usage:_
+ It is electromechanical device and was used to sense the vibration, orientation and acceleration of a bot.
+
+### GYROSCOPE:
+#### _Motto of usage:_
+ Gyroscope was used to determine angular velocity orientation of bot.
+
```
Comment (SanjeevKrishnan) [inline README.md:158]: Add Cost Structure after Electronics Aspect.

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@pabbathisowmya Changes Required
1. Add Videos folder with either CAD stimulation or Arduino code stimulation. "Images" and "Videos" folders have to be inside folder "Images and Videos".
2. Add CAD Images inside Mechanical Design folder.
3. In Poster, Add "Models and Robotics Section", "Srishti 2020", small abstract of your project. Also change MaRS logo with its official logo.
4. Report is not upto the mark. Add a proper front page with "Models and Robotics Section", "Srishti 2020", project name and an index page. Also don't break sub-headings and content into two pages. For example, in your report, heading "Team Members" is in one page and list of team members is in another page.
5. File Arduino code has to be inside a folder named "src". Also rename it to some project specific name. 
And extension should be .ino(not .txt)
6. Changes in README.md are given below

**Comment by pabbathisowmya** [comment]
@SanjeevKrishnan @Shubhanshu07

**Comment by agshubhs** [comment]
@pabbathisowmya PR looks good to me .Merging

**Pull request merged**